docs/UsersGuide/source/BuildingRunningTesting/AQM.rst

```diff
@@ -68,7 +68,7 @@ If the SRW-AQM builds correctly, users should see the standard executables liste
    * - nexus
      - Runs the NOAA Emission and eXchange Unified System (:ref:`NEXUS <nexus>`) emissions processing system
 
-Load the ``workflow_tools`` Environment
+Load the ``srw_app`` Environment
```

Comment:
I might also mention that with the deprecation of supported platform conda environments for SRW, we might want to consider making the conda installation default. Let's plan to bring this up in the code manager meeting since there are lots of pros/cons to each strategy.

Reply:
The result from the developer meeting was to go with conda installed by default with no target. It is turned off by default when building AQM.